filetype.yml

```diff
@@ -0,0 +1,36 @@
+name: marda_filetype_schema
+id: marda_filetype_schema
+default_range: string
+prefixes:
+  schema: http://schema.org
+
+classes:
+  FileType:
+    description: >-
+      A specific encoding of data for storage purposes, defined by a set of common characteristics and expectations, that can be assumed for all files of a given file type.
+    attributes:
+      uuid:
+        identifier: true
+        slot_uri: schema:identifier
+      name:
+        slot_uri: schema:name
+      description:
+        slot_uri: schema:description
+      vendor:
```

Comment:
```suggestion
      primary_vendor:
```
?

```suggestion
   associated_vendors: 
      multivalued: true
````

Reply:
I think `associated_vendors` is perhaps more descriptive.